packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -57,14 +57,22 @@ export const messages = {
   parameter_gatherer_enter_lwc_name:
     'Enter desired Lightning Web Component (Press Enter to confirm or Esc to cancel)',
   parameter_gatherer_enter_username_name: 'Enter target username',
+  parameter_gatherer_enter_source_org:
```

Comment:
cc @ekapner can you look at the new labels and messages for these new commands ?

Reply:
Hey @lcampos - when you say desired, do you mean it exists already or are you entering the name for a new one?